sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClient.java

```diff
@@ -187,6 +225,32 @@ public String getEventHubName() {
             .flatMap(node -> node.getPartitionProperties(partitionId));
     }
 
+    /**
+     * Indicate whether this client is an idempotent producer.
+     * @return Whether this client is an idempotent producer.
+     */
+    public boolean isIdempotentProducer() {
```

Comment:
That's not part of the design.   This is intended to be exposed by the `PartitionPublishingProperties`

Reply:
Removed it from EventHubProducerClient. I'm also wondering whether we need this anywhere because a user knows whether a producer is an idempotent producer. And this is a client-side property.